dot/simswap/util/reverse2original.py

```diff
@@ -149,13 +151,55 @@ def reverse2wholeimage(
                     use_gpu=use_gpu,
                 )
 
-                target_image = cv2.warpAffine(target_image_parsing, mat_rev, orisize)
+                if use_gpu:
+                    target_image = ko_transform.warp_affine(
```

Comment:
you may be able to refactor these conditional branches by this https://pytorch.org/docs/stable/generated/torch.Tensor.to.html#torch-tensor-to

Reply:
that's correct. I would actually define a `device` variable which is set to `"cuda"` or `"cpu"` based on `use_gpu`.

I've added a TODO for this. For now, I tried to maintain the code style of this function.

Reply:
Sure

Reply:
Done